packages/ihe-gateway/ihe-gateway-setup.xml

```diff
@@ -6955,113 +4031,139 @@
+          xF0zGJCb3bMj7sL1MES0OAvtVZk7FOCRjWlHMB85n81QvaU27G650qARoPaai+MAbup2RSOWLp6G&#xd;
+          U00B5fTFIQ8wktnRu6qaI2nQ10ZkI1/jHNkHfBz8jc7ekfA5PvROJbmQiJQEW7+SuFWdwYJQThVJ&#xd;
+          yRPD+mugXlV06mGW5MK4wDY+lmW7uSR4itjJG1mo/bIi8qndX//0MlF/sUezdoA04za9PDkZTwc2&#xd;
+          jLNIM1g+K3CWk4CxVf3Y4SMfTVCCSN6zsXpBoRGU0K+/8+9pfd0R6hvduJsCnoMVJFzGj0ew0MPe&#xd;
+          ziDU8oe8UMmxNotanHmuahzLPua3p2/f6XWdBh7FLuA6vVT7RNjiy90tyyBHo7Qaifad2NhJTTdU&#xd;
+          bwP2UprGFg1GH6Bx9QXr1tiQnDY5hj6lbcpTIwyfM7fYYxdlk83JRrkAAAABAAVYLjUwOQAABSAw&#xd;
+          ggUcMIIEBKADAgECAghRt2veBKZ6HzANBgkqhkiG9w0BAQsFADCBqzELMAkGA1UEBhMCVVMxEzAR&#xd;
+          BgNVBAgMCkNhbGlmb3JuaWExEjAQBgNVBAcMCVNhbiBEaWVnbzETMBEGA1UECgwKRU1SIERpcmVj&#xd;
+          dDE2MDQGA1UECwwtQ2VydGlmaWNhdGlvbiBBdXRob3JpdHkgKGNlcnRzLmVtcmRpcmVjdC5jb20p&#xd;
+          MSYwJAYDVQQDDB1FTVIgRGlyZWN0IFRlc3QgRGV2aWNlIFN1YkNBMjAeFw0yMzExMDIyMTI1NDJa&#xd;
+          Fw0yNDExMDIyMTI1NDJaMHoxCzAJBgNVBAYTAlVTMRMwEQYDVQQIDApDYWxpZm9ybmlhMRcwFQYD&#xd;
+          VQQKDA5NZXRyaXBvcnQgSW5jLjEZMBcGA1UECwwQQ0FSRVFVQUxJVFktVEVTVDEiMCAGA1UEAwwZ&#xd;
+          aWhlLnN0YWdpbmcubWV0cmlwb3J0LmNvbTCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEB&#xd;
+          AOLTmu5aAj4F5314yidZPsFp2/RHWdy1dydQnzuyoOK/9VAxLMoNMzVvw+yqYGSIvW+GFnNmAuy8&#xd;
+          wzZ5NXeLAe+u63TJNHCL+MrH6VzlchhVxeEc6GXBsQVN2YW6icm47bpw65rwPDR8pg+Af289vLCF&#xd;
+          UbSQ1dB/4OCO4wzcgpeiFHkCSlEwWvdFA3SYMRZzA9D5rsYwONIp442e9OxrgPkrYIANJBgiH4UB&#xd;
+          FZ9YO4i9zvM6PWKdMt0JFrQYK8Tk5yP9VD4+kqtKgZQ9QtKG4d9uziTjFPai0ayS7OjCFM8uz3mS&#xd;
+          8YJUGwil+c0YbRA3tRvxakMP+c3MYnAbIF7XpTECAwEAAaOCAXIwggFuMFoGCCsGAQUFBwEBBE4w&#xd;
+          TDBKBggrBgEFBQcwAoY+aHR0cDovL2NlcnRzLmVtcmRpcmVjdC5jb20vY2VydHMvRU1SRGlyZWN0&#xd;
+          VGVzdERldmljZVN1YkNBMi5jcnQwHQYDVR0OBBYEFMeEfSqQ5K65vH2mMdoKGyCMF98lMAwGA1Ud&#xd;
+          EwEB/wQCMAAwHwYDVR0jBBgwFoAUsAtYO8WTrcZvZV0RwBKtWtyHhjUwTQYDVR0fBEYwRDBCoECg&#xd;
+          PoY8aHR0cDovL2NlcnRzLmVtcmRpcmVjdC5jb20vY3JsL0VNUkRpcmVjdFRlc3REZXZpY2VTdWJD&#xd;
+          QTIuY3JsMA4GA1UdDwEB/wQEAwIFoDAdBgNVHSUEFjAUBggrBgEFBQcDAQYIKwYBBQUHAwIwRAYD&#xd;
+          VR0RBD0wO4IZaWhlLnN0YWdpbmcubWV0cmlwb3J0LmNvbYYeSFRUUDovL1dXVy5DQVJFUVVBTElU&#xd;
+          WS5PUkcvVjAxMA0GCSqGSIb3DQEBCwUAA4IBAQAIBJRMmH0pCF2KSy97WxHxxP9sHlGfXaq+Wjfq&#xd;
+          Q1/QW8rqrFRj5t6LcXXF5nZf7YAuLzwTKk/Bcf03QIWxQ0sbu3CeNW24gQ+r+7o1Th1fnld0IhFB&#xd;
+          xOOPpy1Isag3Vo2jaMEP8W09Jy5NH3lz8bNrjdFOolo8qtWdpZ31ADpNVIGKYih2lgPPTZyIza/z&#xd;
+          z2Irweph4aCTebpKYDKWJFA1wzUIzR2eObcnBrZ2Edi0013RRRJ3eYcC6o5PL2D0oEWnqlRVxVho&#xd;
+          Nj2bSD2GYQEmoBHhRrc9i51UCfmJuK7jl/Fe9+Dx7CH2++LD8U9+OojASI0DncHbgY+eQL0n+fZL&#xd;
+          sp+a/SMQaIewLToqL9gPnVng29k=&#xd;
+        </property>
+        <property name="KeyStorePassword">1648fb07-37d8-411a-9412-2734b06b764f</property>
+        <property name="TrustStore">/u3+7QAAAAIAAAADAAAAAgASZW1yIGRpcmVjdCB0ZXN0IGNhAAABjBEv8pgABVguNTA5AAAGajCC&#xd;
```

Comment:
why is the trust store way bigger now? this shouldn't be changing with this backup